test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -58,6 +58,12 @@ public void listCommand_parsedCorrectly() {
         parseAndAssertCommandType(input, ListCommand.class);
     }
 
+    @Test
+    public void LastCommand_parsedCorrectly() {
```

Comment:
Try to work on adding multiple test cases in the future. e.g. user type last without any last command available